lib/src/main/java/dataframe/Row.java

```diff
@@ -65,6 +65,9 @@ public boolean getBoolean(int i) {
     public String getString(int i) {
         return this.getAs(i);
     }
+    public String getString(String column) {
+        return this.data.get(this.fieldIndex(column)).toString();
```

Comment:
use `String.valueOf` instead of `toString` to avoid `NullPointerException`

Reply:
Done